LICENSE.BINARY

```diff
@@ -794,176 +794,170 @@ BINARY/EXTENSIONS/druid-kafka-eight
 BINARY/WEB-CONSOLE
 The following dependency names are NPM package names (https://www.npmjs.com).
 
-    This product bundles @babel/runtime version 7.3.4, copyright Sebastian McKenzie and other contributors,
+    This product bundles @babel/runtime version 7.4.3, copyright Sebastian McKenzie,
      which is available under an MIT license. For details, see licenses/bin/@babel-runtime.MIT.
 
+    This product bundles @types/dom4 version 2.0.1, copyright ,
```

Comment:
There should be a copyright holder here. If the holder is unknown or generic, then remove the `copyright ,` string.

Reply:
Thanks, fixed others as well.